praw/models/reddit/live.py

```diff
@@ -281,6 +281,58 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def update(self, title=None, description=None, nsfw=None, resources=None,
+               **other_settings):
+        """Update settings of the live thread.
+
+        :param title: (Optional) The title of the live thread (default: None).
+        :param description: (Optional) The live thread's description
+            (default: None).
+        :param nsfw: (Optional) Indicate whether this thread is not safe for
+            work (default: None).
+        :param resources: (Optional) Markdown formatted information that is
+            useful for the live thread (default: None).
+
+        Return ``None`` if there is no argument.
```

Comment:
Since `None` isn't explicitly returned, can we instead say something like "Does nothing if no arguments are provided?